src/analysis/ContextBar.test.ts

```diff
@@ -513,19 +485,22 @@ describe('ContextBar - buttons', () => {
     expect(getByLabelText(new RegExp(/RStudio Environment/i)));
     expect(getByLabelText(new RegExp(/Galaxy Environment/i)));
     expect(queryByTestId('terminal-button-id')).not.toBeInTheDocument();
-    expect(getByText(/Running \$.*\/hr/));
-    expect(getByText(/Creating \$.*\/hr/));
-    expect(getByText(/Disk \$.*\/hr/));
+    expect(getByText('Running $0.52/hr'));
+    expect(getByText('Creating $0.20/hr'));
+    expect(getByText('Disk $0.04/hr'));
```

Comment:
`getBy...` queries don't need to be wrapped in expect since they will throw an error if no match is found.